### PR TITLE
[12.0] account_multicurrency_revaluation: Fixes and improvements

### PR DESCRIPTION
**Pull request by grindtildeath**
Forward port of #101

### CODE REVIEW

File: account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py [hunk 1]
```diff
@@ -132,18 +132,18 @@ def _compute_unrealized_currency_gl(self, currency_id, balances, form):
         # Compute unrealized gain loss
         ctx_rate = context.copy()
         ctx_rate['date'] = form.revaluation_date
+        ctx_rate['revaluation'] = True
```
Comment (grindtildeath) [inline account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py]: @hbto I did move this context here, but is it really needed ? :thinking:

Comment (grindtildeath) [reply account_multicurrency_revaluation/wizard/wizard_currency_revaluation.py]: Resolved in a32ce49

**Review by i-vyshnevska — APPROVED** [review]
LG

**Comment by yvaucher** [comment]
@grindtildeath can you rebase, please?

**Comment by grindtildeath** [comment]
@yvaucher done :smiley:

**Review by yvaucher — APPROVED** [review]

**Pull request merged**